schemas/entry.schema.json

```diff
@@ -80,11 +80,68 @@
       "additionalProperties": false,
       "properties": {
         "breakage": {
-          "description": "List of URLs to issues where actual site breakage is tracked. If available, point the link directly to a comment with clear STR",
+          "description": "List of issues tracking known site breakage.",
           "type": "array",
           "items": {
-            "type": "string",
-            "format": "uri"
+            "anyOf": [
+              {
+              "type": "string",
+              "format": "uri",
+              "description": "URL of tracking issue"
+            },
+              {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of tracking issue"
+                },
+                "site": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of broken site of page"
+                },
+                "platform": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": ["all", "desktop", "mobile", "windows", "macos", "linux"],
+                    "description": "List of affected platforms. Default is 'all'"
+                  }
+                },
+                "last_reproduced": {
+                  "type": "string",
+                  "format": "date",
+                  "description": "Most recent date the issue was successfully reproduced"
+                },
+                "intervention": {
```

Comment:
Do we ever have > 1 intervention per site?

Reply:
For the "same" issue that is; I imagine we could have a site affected by multiple kinds of breakage that get seperate interventions.

Reply:
While >1 intervention per site is possible in theory (hasn't happened yet), more than one intervention per site *per issue* is unlikely.

Reply:
I think there is a google maps intervention that has css and js files
https://github.com/mozilla-extensions/webcompat-addon/blob/main/src/data/injections.js#L170

Reply:
yeah, but that's "the same" intervention, and we should probably link to the definition, instead of the individual files (in that case? in general?).

Reply:
True, and I guess in this case the js intervention is the more important one.. could either link to it, or the definition. 

In general, it seems individual files would be easier to locate, if the intervention is removed (
provided we use canonical urls 🙂), like you mentioned in https://github.com/webcompat/knowledge-base/pull/70#discussion_r927828547